### DISCUSSION TITLE
How can I get the sidebar to render on the home page?

### DISCUSSION THREAD

**Discussion by CodyBontecou** [vuejs/vitepress#1010] [Q&A] [answered]
Is this a config variable I can pass to my homepage to also render the sidebar?

**Comment by brc-dd** [comment, marked as answer]
It was intentionally removed. Refer https://github.com/vuejs/vitepress/issues/825#issuecomment-1161551559

**Comment by johnson86tw** [comment]
Hey, this should be optional. I like the sidebar rendering on the home page instead of creating a useless landing page..
